### ISSUE TITLE
Example for awful.placement.no_overlap is hard to understand

### ISSUE THREAD

**Issue by psychon** [awesomeWM/awesome#1332]
https://awesomewm.org/apidoc/images/AUTOGEN_awful_placement_no_overlap.svg

What is this trying to tell me? What's up with the two empty screens in-between? This is not really `no_overlap`, because `no_overlap` does not resize clients, it only moves them. No idea what is going on, but the example is wrong.

**Comment by Elv13** [comment]
Yes, this will need some work. The client moving to the wrong screen is a bug. It was already there when I wrote these examples. There is some others bugs in the module I never fixed (predating my contributions). I will do it eventually
